Centroids[] data
#True for empty list, False means not empty
bool empty_or_not
# number of matched pedestrians
int16   num_of_matches
float64 sum_of_distance

